commit_verify/src/embed.rs

```diff
@@ -303,8 +293,8 @@ mod test {
     impl<T> EmbedCommitProof<T, DummyVec, TestProtocol> for DummyProof
     where T: AsRef<[u8]> + Clone + CommitEncode
     {
-        fn restore_original_container(&self, _: &DummyVec) -> Result<DummyVec, Error> {
-            Ok(DummyVec(self.0.clone()))
+        fn restore_original_container(&self, _: &DummyVec) -> Option<DummyVec> {
```

Comment:
Why change from Result to Option?

Reply:
This is internal method called from verify, which was relying on `type VerifyError` which is not there anymore. So there is no error type we can return from here.